.gitignore

```diff
@@ -102,3 +102,4 @@ cloudinary/static/
 log/
 output.html
 .idea
+.vscode
```

Comment:
Please make a separate commit for this change (it is not related to this PR)
And put a newline at the end of file